TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Administration/PackagesController.php

```diff
@@ -44,7 +44,7 @@ public function indexAction()
             $packageGroup = substr($packagePath, 0, strpos($packagePath, '/'));
             $packageGroups[$packageGroup][$package->getPackageKey()] = array(
                 'sanitizedPackageKey' => str_replace('.', '', $package->getPackageKey()),
-                'version' => $package->getPackageMetaData()->getVersion(),
+                'version' => $package->getInstalledVersion(),
```

Comment:
I can't seem to find that used method anywhere in the Package class.

A extra note, the annotation about $package being a \TYPO3\Flow\Package instance, is wrong. It's actually a \TYPO3\Flow\Package\Package instance